### ISSUE TITLE
Deploy automatically?

### ISSUE THREAD

**Issue by jacebrowning** [Ross-Hunter/nbp-site#6]
Is there any reason we shouldn't do this?

**Comment by Ross-Hunter** [comment]
I do not believe there is any reason not deploy from master. The template I chose came with a deploy script, and I haven't set up any delivery out of laziness.
